src/ProjectSystemSetup/ProjectSystemSetup.csproj

```diff
@@ -7,8 +7,6 @@
     
     <!-- VSIX -->
     <IncludeAssemblyInVSIXContainer>false</IncludeAssemblyInVSIXContainer>
-    <ExtensionInstallationRoot>CommonExtensions</ExtensionInstallationRoot>
-    <ExtensionInstallationFolder>Microsoft\VisualStudio\Editors</ExtensionInstallationFolder>
```

Comment:
Where is this installed now? The project system did not used to go in the Editors folder but I think we still want it in a well-known location right?

Reply:
@jmarolf It ends up in a folder under Common7\IDE\Extensions\ where it used to be. I've confirmed through installing a VAL build that this fixes the missing icon issues in d15prerel. /cc @MattGertz for Ask Mode since Kevin is out.

Reply:
I now would like an explanation as to *why* this fixes the issue.  Where do we load these icons from that they can only be found if they are places in `Common7\IDE\Extensions`?  That seems like an important implicit contract that we need to understand.

Reply:
I suspect the location doesn't matter -- placing the two components in the same folder was having the effect of clobbering some files. I see `manifest.json`, `catalog.json` and some other files being clobbered -- though it is not clear how they relate to overall configuration of the extension and why other features are not broken. We could test if location matters by doing another VAL build, but that probably doesn't need to make the snap for Preview 1.